core-libs/setup/ssr/logger/loggers/express-server-logger.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { InjectionToken } from '@angular/core';
+import { Request } from 'express';
+import { SsrOptimizationOptions } from '../../optimized-engine';
+
+export interface ExpressServerLoggerContext {
+  request: Request;
+  options?: Partial<SsrOptimizationOptions>;
+}
```

Comment:
[MEDIUM]

IMHO only the `request: Request` is specific to the ExpressJS (and therefore to. `ExpressServerLoggerContext`)
I don't think we should couple it specifically with `options` and `SsrOptimizationOptions`.

Btw. I think this interface should: (1) require `request` and (2) allow for any context. What do you think?
```suggestion
export interface ExpressServerLoggerContext {
  request: Request;
  [_key: string]: any;
}
```